src/oscar/apps/basket/forms.py

```diff
@@ -61,13 +61,19 @@ def check_max_allowed_quantity(self, qty):
         # add second time, just updating number of items.
         qty_delta = qty - self.instance.quantity
         is_allowed, reason = self.instance.basket.is_quantity_allowed(qty_delta)
+        if self.instance.basket.strategy.request and self.instance.basket.strategy.request.is_ajax:
```

Comment:
`is_ajax` is a method, not a property. The callable itself will always evaluate to truthy.